api/helpers/errors.py

```diff
@@ -0,0 +1,63 @@
+class InternalServerError(Exception):
+    pass
+class SchemaValidationError(Exception):
+    pass
+class EducationAlreadyExistsError(Exception):
+    pass
```

Comment:
I have not seen this syntax before, but looks like this is just a helper file for what error message to return?

Reply:
Yes!  That's exactly what it is.  We can customize our error changes.

Reply:
This represents a helper/errors.py file (that still needs updates: it isn't working, but we will be writing sad paths in future PR's and ensuring error handling is working as expected).
Yes - it simply holds information for custom errors